src/components/form/SignUp.tsx

```diff
@@ -0,0 +1,45 @@
+import { FC } from "react";
+
+import { useForm, SubmitHandler } from "react-hook-form";
+import { SignUpFormData, } from "../../types/Types";
+import FormInputComponets from "./BaseInput";
+import BaseButton from "../button/BaseButton";
+
+const SignUp: FC = () => {
+
+    const { register, handleSubmit, formState: { errors } } = useForm<SignUpFormData>();
+
+    const onSubmit: SubmitHandler<SignUpFormData> = (data) => {
+        alert(`Your name ${data.name}`)
+    }
+
+    return (
+        <div className="flex justify-center mt-16">
+            <div className=" w-1/3 space-y-8  bg-white  p-14">
```

Comment:
the same here. problem with two divs

You can create a registration wrapper for those forms and reuse this component wrapper for both sign up and sign in components